### PR TITLE
[stable/zetcd] Added nodeSelector for zetcd.

### PR DESCRIPTION
**Pull request by hchenxa**
[stable/zetcd] Added nodeSelector for zetcd.
related to https://github.com/kubernetes/charts/issues/1899

### CODE REVIEW

File: stable/zetcd/values.yaml [hunk 1]
```diff
@@ -24,10 +24,12 @@ resources: {}
   #  cpu: 100m
   #  memory: 128Mi
 
+nodeSelector: {}
```
Comment (gyliu513) [inline stable/zetcd/values.yaml]: This is not enough, you need to add this to deployment, check here for detail https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/templates/deployment.yaml#L50-L53

Also please add some comments here, take here as reference https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/values.yaml#L15-L18

**Comment by gyliu513** [comment]
I'd prefer that the summary can be updated to

```
[stable/zetcd] Added nodeSelector for zetcd.
```

Also update the version here https://github.com/kubernetes/charts/blob/master/stable/zetcd/Chart.yaml#L4 to `0.1.3`

File: stable/zetcd/values.yaml (hunk 1 above)
Comment (hchenxa) [reply stable/zetcd/values.yaml]: no need update the deployment as the deployment has already included the nodeSelector section:
https://github.com/kubernetes/charts/blob/aa0380f852fb8d71d57acc8f63306699adfb827f/stable/zetcd/templates/deployment.yaml#L42-L44

Comment (gyliu513) [reply stable/zetcd/values.yaml]: I see, then it is better add some comments here as https://github.com/kubernetes/charts/blob/master/stable/aws-cluster-autoscaler/values.yaml#L15-L18

**Comment by gyliu513** [comment]
Also need to update the commit message to 

```
[stable/zetcd] Added nodeSelector for zetcd.
```

File: stable/zetcd/values.yaml (hunk 1 above)
Comment (hchenxa) [reply stable/zetcd/values.yaml]: already have comments.

**Comment by gyliu513** [comment]
Any feedback from chart maintainers? Can we have this got approved? /cc @kubernetes/charts-maintainers

**Comment by mattfarina** [comment]
Since nodeSelector is already in the deployment for this chart it lgtm but... I'm not a charts maintainer so one of them needs to chime in.

**Pull request merged**